shaky/src/main/java/com/linkedin/android/shaky/Shaky.java

```diff
@@ -240,6 +291,10 @@ public void onDataReady(@Nullable Result result) {
 
                 if (shouldStartFeedbackActivity) {
                     startFeedbackActivity(result == null ? new Result() : result);
+                } else {
+                    if (shakyFlowCallback != null) {
+                        shakyFlowCallback.onShakyFinished(ShakyFlowCallback.SHAKY_FINISHED_NO_RESUMED_ACTIVITY);
+                    }
```

Comment:
I would prefer to have a return after startFeedbackActivity and remove the nested if here